### PR TITLE
Fix `UnusedBlockArgument` and `UnusedMethodArgument` offenses reported by rubocop

### PR DESCRIPTION
**Pull request by tejasbubane**

### CODE REVIEW

File: lib/app/presenters/workload.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class NullWorkload
-  def next_submission(submission)
+  def next_submission(_submission)
```
Comment (kytrinyx) [inline lib/app/presenters/workload.rb]: Let's just call it `_`.

File: lib/app/routes/metadata.rb [hunk 2]
```diff
@@ -10,7 +10,7 @@ class Metadata < Core
         end
         data = data["assignments"].first
         problem = Problem.new(data['track'], data['slug'])
-        text = data["files"].find {|filename, code|
+        text = data["files"].find {|filename, _code|
```
Comment (kytrinyx) [inline lib/app/routes/metadata.rb]: same here. No need to name it if we're ignoring it. `_` should be fine.

**Comment by kytrinyx** [comment]
For unused variables/arguments/params I like to use `_` rather than naming them.

**Comment by tejasbubane** [comment]
@kytrinyx done.

**Comment by kytrinyx** [comment]
I'm getting merge conflicts. Would you rebase this onto master? (I can resolve this locally if you don't have time).

**Comment by tejasbubane** [comment]
@kytrinyx rebased on master.

**Comment by kytrinyx** [comment]
Sweet, thanks!

**Pull request merged**